samples/bluetooth/df_per_adv_beacon/README.rst

```diff
@@ -0,0 +1,79 @@
+.. _bluetooth_df_per_adv_beacon:
+
+Bluetooth: Direction Finding Periodic Advertising Beacon
+########################################################
+
+Overview
+********
+
+A simple application demonstrating the BLE Direction Finding CTE Broadcaster
+functionality by sending Constant Tone Extension with periodic advertising PDUs.
+
+
+Requirements
+************
+
+* nRF52833DK board with nRF52833 SOC
+* antenna matrix for AoD (optional)
+
+Building and Running
+********************
+
+This sample can be found under :zephyr_file:`samples/bluetooth/df_per_adv_beacon`
+in the Zephyr tree.
+
+By default the application supports Angle of Arrival and Angle of Departure
+mode.
+
+To use Angle of Arrival mode only, build this application as follows:
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/bluetooth/df_cl_beacon
```

Comment:
Should be:
```suggestion
   :zephyr-app: samples/bluetooth/df_per_adv_beacon
```
if i'm not mistaken.

Reply:
You are right, thank you for spotting that. I've already corrected it.